### PR TITLE
[New Model] Week-Based Capacity Group (Demand and Capacity Management)

### PR DESCRIPTION
**Pull request by LukasHeimann**
## Description
This aspect contains data about a Capacity Group of a given supplier for a particular customer. It is necessary for the use case of Demand and Capacity Management (DCM), representing the "Capacity" part of the use case. A parallel request has been opened for the corresponding model of a Material Demand as shared by the customer (https://github.com/eclipse-tractusx/sldt-semantic-models/pull/92). Both models co-evolve.

In DCM, customers along a supply chain want to make the demands for their materials at certain locations with certain demand categories available for their suppliers. Demand data is provided on a weekly basis for the next two years. Suppliers, in return make their capacities for certain demand series (based on material, location and demand category) transparent to the respective customers.

Compare
- https://github.com/catenax/BAMMmodels/issues/183
- https://github.com/catenax/BAMMmodels/pull/208

## MS2 Criteria
(to be filled out by PR reviewer)
- [x] the model **validates** with the BAMM SDS SDK in the version specified in the Readme.md of this repository by the time of the MS2 check  (e.g., 'java -jar bamm-cli.jar -i \<path-to-aspect-model\> -v ). The  BAMM CLI is available [here](https://openmanufacturingplatform.github.io/sds-documentation/sds-developer-guide/dev-snapshot/tooling-guide/bamm-cli.html) and in [GitHub](https://github.com/OpenManufacturingPlatform/sds-sdk/releases)
- [x] use **Camel-Case** (e.g., "MyModelElement" or "TimeDifferenceGmtId", when in doubt follow https://google.github.io/styleguide/javaguide.html#s5.3-camel-case)
- [x] the identifiers for all model elements **start with a capital letter** except for properties
- [x] the identifier for **properties starts with a small letter**
- [x] all model elements **at least contain the fields "name" and "description"** in English language. 
- [x] **no duplicate names or preferredNames** within an Aspect (e.g. a Property and the referenced Characteristic should not have the same name)
- [x] the versioning in the URN **follows semantic versioning**, where minor version bumps are backwards compatible and major version bumps are not backwards compatible. 
- [x] use **abbreviations only when necessary** and if these are sufficiently common
- [x] **avoid redundant prefixes in property names** (consider adding properties to an enclosing Entity or even adapt the namespace of the model elements, e.g., instead of having two properties `DismantlerId` and `DismantlerName` use an Entity `Dismantler` with the properties `name` and `id` or use a URN like `io.catenax.dismantler:0.0.1`)
- [x] fields `preferredName` and `description` are not the same
- [x] **`preferredName` should be human readable** and follow normal orthography (e.g., no camel case but normal word separation)
- [x] name of aspect is singular except if it only has one property which is a Collection, List or Set. In theses cases, the aspect name is plural.
- [x] units are referenced from the BAMM unit catalog whenever possible
- [x] **use constraints** to make known constraints from the use case explicit in the aspect model 
- [x] when relying on **external standards**, they are referenced through a **"see"** element
- [x] all properties with an [simple type](https://openmanufacturingplatform.github.io/sds-documentation/bamm-specification/v1.0.0/datatypes.html) have an example value
- [x] metadata.json exists with status "release"
- [x] file RELEASE_NOTES.md exists and contains entries for proposed model changes 

## MS3 Criteria
(to be filled out by semantic modeling team before merge to main-branch)
- [x] All required reviewers have approved this PR (see reviewers section)
- [x] The new aspect (version) will be implemented by at least one data provider
- [x] The new aspect (version) will be consumed by at least one data consumer
- [x] There exists valid test data
- [x] In case of a new (incompatible) major version to an existing version, a migration strategy has been developed
- [x] The model has at least version '1.0.0'

### CODE REVIEW

**Review by bs-jokri — APPROVED** [review]
LGTM

**Comment by bs-jokri** [comment]
same comment as here: https://github.com/eclipse-tractusx/sldt-semantic-models/pull/92#issuecomment-1451670206

File: io.catenax.week_based_capacity_group/1.0.0/WeekBasedCapacityGroup.ttl [hunk 1]
```diff
@@ -0,0 +1,270 @@
+#######################################################################
+# Copyright (c) 2022 SAP SE
```
Comment (catroest) [inline io.catenax.week_based_capacity_group/1.0.0/WeekBasedCapacityGroup.ttl:2]: 2023?

**Comment by petyog** [comment]
@bs-jokri are there still any open tasks / pending checks or can this PR be merged? We need the final link to the `WeekBasedCapacityGroup.ttl` file for the corresponding standardization request (CX-0047).

**Comment by bs-jokri** [comment]
@petyog automatic tests are completed and PR is merged. You can find the model in the main branch

**Pull request merged**